core-web/libs/sdk/angular/src/lib/models/dotcms.model.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 export interface DotCMSContainer {
```

Comment:
I know is a type, but could be nice a doc about what is this. Same for the other models.